### DISCUSSION TITLE
It would be nice to get rid of UWP app icon backplates

### DISCUSSION THREAD

**Discussion by AzAgarampur** [File-New-Project/EarTrumpet#720] [Ideas]
It would be cool to drop the backplates behind the icons for UWP apps in EarTrumpet, the same way that the new Start Menu has done. Here's what I mean:

![image](https://user-images.githubusercontent.com/42058942/100946568-7f578100-34b8-11eb-90d5-acecc09740d6.png)

Notice how the icons in the Start Menu have no backplate of the current accent color, but in EarTrumpet they do:

![image](https://user-images.githubusercontent.com/42058942/100946601-9b5b2280-34b8-11eb-8b7c-2d3612872f28.png)

**Comment by riverar** [comment]
Agreed. At least for Windows 10 October 2020 Update and newer.
